src/game.cpp

```diff
@@ -3188,7 +3191,7 @@ void game::debug()
 #if defined(TILES)
             const point offset {
                 POSX - u.posx() + u.view_offset.x,
-                     POSY - u.posy() + u.view_offset.y
+                POSY - u.posy() + u.view_offset.y
```

Comment:
Astyle on the CI servers doesn't like this for some unknown reason.  See #29465
```suggestion
                    POSY - u.posy() + u.view_offset.y
```

Reply:
I'm using Astyle 3.1 at the moment. I guess it's related to #29465.